data/base-contacts.yaml

```diff
@@ -0,0 +1,3356 @@
+contacts:
```

Comment:
This file is created using all of the information in the "of renown" charts for each of the guilds on elanthipedia. As such, I'd consider all of it "public information" and planned to have it be the base file everyone got on install to give them some benefit right away. The community could also choose to continue adding more information if desired. It might make more sense to remove the lines for relationship, notes, and alts, since I don't foresee those ever being used in the public file, but I haven't tested how the scripts respond to those being missing.

There would also be a personal file yourname-contacts.yaml in the profiles folder. This is where each player could record personal notes or add information on people that they didn't wish to share with the entire lich community.